index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+
+    <title>Setup and mobile version skeleton</title>
+</head>
+<body>
+  <nav>
+    <ul>
+      <li id="logo"><a href="#"><img src="images/John-Doe.png" alt="Website logo"></a></li>
+      <li id ="menu"><a href="#"><img src="images/Icons/Menu.png" alt="Website menu"></a></li>
+    </ul>
+  </nav>
+<header>
+
+    <h1>Hey There.
+  I’m Alex</h1>
+    <h3>I’am a Software Developer</h3>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+  don’t hestiate to contact me.</p>
+
```

Comment:
- [ ] Kindly align the text content here to the left, currently, in your design, it is aligned to the center